### DISCUSSION TITLE
Using wg-easy with an existing wireguard server / mitmproxy

### DISCUSSION THREAD

**Discussion by eliasto** [wg-easy/wg-easy#1508] [Q&A]
Hello everyone!
I use `mitmproxy` to proxy my requests and modify them, using the wireguard mode. This mode creates its own server, with a user. As soon as I connect to this user with WireGuard, I see my requests.
I would like to use `wg-easy` to be able to create several users simply.
Unfortunately, I have a hard time doing it. Do you know how it is possible to modify the wg-easy config, to use the mitmproxy wireguard server instead?

Thank you very much.
Regards,
Elias.

**Comment by kaaax0815** [comment]
i am not familiar with mitmproxy but this isn't supported by wg-easy
